jspwiki-event/pom.xml

```diff
@@ -45,6 +45,12 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
I'd rather not use a constant for blank spaces, slashes or other types of constants where there is no semantic meaning into them. Using these kind of constants only couples two modules b/c of a constant (or brings in commons-lang) for the sake of reusing a constant. 

In this case, jspwiki-event will bring jspwiki-utils and all of its transitive dependencies, just for a constant, that is going to be translated to its value by the compiler on compile time. It'd be different if the constant would be referring to, i.e., a specific key of the jspwiki.properties file, then it would make sense to me

WDYT? (and what do others think about this?)